doc/source/PreConfiguredSites.rst

```diff
@@ -464,11 +571,36 @@ For ``spack-stack-1.3.0`` with Intel, run the following commands/load the follow
    module load stack-python/3.10.8
    module available
 
+For ``spack-stack-1.3.1`` with Intel, run the following commands/load the following modules:
+
+.. code-block:: console
+
+   module purge
+   ulimit -s unlimited
+   source /opt/intel/oneapi/compiler/2022.1.0/env/vars.sh
+   module use /mnt/experiments-efs/skylab-v4/spack-stack-1.3.1/envs/unified-env/install/modulefiles/Core
+   module load stack-intel/2022.1.0
+   module load stack-intel-oneapi-mpi/2021.6.0
+   module load stack-python/3.10.8
+   module available
+
+For ``spack-stack-1.3.1`` with GNU, run the following commands/load the following modules:
+
+   module purge
+   ulimit -s unlimited
+   module use /mnt/experiments-efs/skylab-v4/spack-stack-1.3.1/envs/unified-env/install/modulefiles/Core
+   module load stack-gcc/9.4.0
+   module load stack-openmpi/4.1.4
+   module load stack-python/3.10.8
+   module available
```

Comment:
The only addition I would suggest (not sure that it goes here), is that I have to load one of the env modules, such as `module load jedi-fv3-env/unified-dev`  (at least on S4 this was necessary).

Reply:
Correct, that is part of jedi-docs. spack-stack documentation needs to be independent of the application (ufs-weather-model, global-workflow, skylab, ...) therefore we didn't add those module load commands here.